utils/getExchangeLinks.ts

```diff
@@ -0,0 +1,60 @@
+interface TokenExchangeMap {
+  [tokenSymbol: string]: number;
+}
+
+interface ExchangeNameMap {
+  [exchangeType: number]: string;
+}
+
+export enum EXCHANGES {
+  UNISWAP,
+  BALANCER,
+}
+
+export const EXCHANGE_NAMES: ExchangeNameMap = {
+  [EXCHANGES.UNISWAP]: "Uniswap",
+  [EXCHANGES.BALANCER]: "Balancer",
+};
+
+// Returns a function for a given exchange that can be used to retrieve an exchange's specific swap URL for a chosen token.
+interface ExchangeLinkMap {
+  [exchangeEnum: number]: (params: any) => string;
+}
+
+// A mapping of synthetic tokens to the exchange where they can be traded.
+export const TOKEN_TO_EXCHANGE_MAP: TokenExchangeMap = {
```

Comment:
Do all of these need to be exported? Might be a little simpler for readers/importers if only the functions intended to be used externally are exported? This may also be a ts or js stylistic thing that I'm missing, though.

Reply:
Nope I meant to remove these. Only `getExchangeInfo` should be exported